3574/3/places/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,34 @@
+# :reek:InstanceVariableAssumption
+class ApplicationController < Sinatra::Base
+  configure do
+    set :views, 'app/views'
+    enable :sessions
+    set :session_secret, 'secret session'
+    register Sinatra::Flash
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  helpers do
+    def current_user
+      @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+    end
+
+    def create_comment
+      @user ||= User.find_by(id: session[:user_id])
+      @comment ||= @user.comments.create(text: params[:text], rating: params[:rating].to_i)
+      @comment.restaurant_id = params[:id]
+      valid_comment
+    end
+
+    def valid_comment
```

Comment:
bad method name, bad usage of method. looks like you are trying to avoid reek/rubocop errors instead of splitting code to readable parts